### ISSUE TITLE
Re-sizing to 200% test is dependent on screen size.  

### ISSUE THREAD

**Issue by brittontripp** [Section508Coordinators/DHSA-TT-200-A#496]
## Issue Type (put a capital X between the brackets to indicate issue type)
- [x] System Bug (the course doesn't function properly)
- [ ] Typo/grammar
- [ ] Incorrect content
- [ ] 508 issue (something is not accessible according to our own test process)
- [ ] Other, specify...[replace this text with your issue type description]

## Description
The test for this page: 

https://exams.section508testing.net/dhsv5/dhscourse/Lesson1/exams/web-incremental/testPages/index.php?studentEmail=britton.tripp%40participate.com&SCOInstanceID=92928&examId=18

Claims that the transcript link gets cut off at 200% zoom.  This is not true if you have a larger screen.  It only gets cut off below a certain browser window size, otherwise it is fine, which leads to an incorrect answer (because it looks fine to me on my screen).  

You either need to fix it to always be cut off at 200% zoom no matter the screen width, or give us a setting for px width of screen so that we know what is appropriate.  

## Location
[[URL, Topic, Chapter, Subchapter]](https://exams.section508testing.net/dhsv5/dhscourse/Lesson1/exams/web-incremental/testPages/index.php?studentEmail=britton.tripp%40participate.com&SCOInstanceID=92928&examId=18)

## Actual Behavior / Observation
My observation is that at 200% zoom, the transcript is fine (not cut off).  The test says otherwise.  

## Expected Behavior / Recommendation
Give us a measurement for width of the browser that it should not respond properly at (responsive site width).  Otherwise make it consistently cut off at any browser window width.  

## Steps to Reproduce the Issue (if applicable)
1.  Open the transcript link on the page above on a 27inch monitor at full screen and zoom to 200% on your monitor. 
2.  Observe that the text is not cutoff.  
3.  Repeat 1-2 on a smaller screen or on the same screen without being at full browser window width.  
4.  Observe that you get a different result dependent on screen size.

**Comment by kahlkevin** [comment]
Absolutely. I encountered this same issue. It's apparent that the exam source code is out of date or insufficient to force the failure condition the examiners intend. I had to just read the explanatory text when I failed, then re-take the test and lie to it (hoping that it didn't randomly generate some other version of the test site with a different failure I also couldn't see...). Frustrating. But not as frustrating as 4.F. Sheesh.
